mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule Benchee.Mixfile do
     [
       app: :benchee,
       version: @version,
-      elixir: "~> 1.6",
+      elixir: "~> 1.7",
```

Comment:
do we absolutely need to do this? I'd like to avoid it. Imo that's kinda part of SemVer so for me if we change support here I'd wanna do a major version bump here. Maybe I'm too strict though.

Reply:
No, not absolutely needed, but Elixir 1.6 is no longer supported in any way, and so I think it makes sense to keep up with their support schedule instead of us supporting Elixir versions that are unsupported.

I totally wouldn't consider that a major version bump (personally), or even a minor version bump. We can always keep it at 1.6 and just say that 1.6 is unsupported.